parsers/verilog.c

```diff
@@ -1577,8 +1580,20 @@ static int tagNameList (tokenInfo* token, int c)
 	{
 		c = processType(token, c, &kind);	// update token and kind
 
-		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`')
+		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`' || c == '(')
 		{
+			if(c == '('){
+					token->kind = K_INSTANCE;
+					kind = K_INSTANCE;
+					c = skipPastMatch ("()");
+					if(c != ';'){
+							token->kind = K_UNDEFINED;
+							kind = K_UNDEFINED;
+					}
+					else {
```

Comment:
we don't use this style, putting { at the end of line IN THIS FILE.

else
{
   ...

is expected.